Clarify split function docs (#1792)
gohugoio/hugoDocs: content/en/functions/split.md

---
title: split
# linktitle: split

> description: splits a string into a slice/array by a delimiter

Reviewer: ```suggestion
description: Returns an array of strings by splitting STRING by DELIMETER.
```

aliases: []
---

> * `{{split "tag1,tag2,tag3" "," }}` → ["tag1", "tag2", "tag3"]

Reviewer: ```suggestion
* `{{ split "tag1,tag2,tag3" "," }}` → ["tag1", "tag2", "tag3"]
```
